aws/terminator/paas.py

```diff
@@ -143,6 +143,60 @@ def terminate(self):
         self.client.delete_cloud_front_origin_access_identity(Id=self.name, IfMatch=self.id)
 
 
+class CloudFrontCachePolicy(DbTerminator):
+    @staticmethod
+    def create(credentials):
+        def list_cloud_front_cache_policies(client):
+            identities = []
+            result = client.get_paginator('list_cache_policies').paginate(
+                # Only retrieve the custom policies
+                Type='custom'
+            ).build_full_result()
+            for identity in result.get('CachePolicyList', {}).get('Items', []):
+                identities.append(client.get_cache_policy(Id=identity['Id']))
+            return identities
+
+        return Terminator._create(credentials, CloudFrontCachePolicy, 'cloudfront', list_cloud_front_cache_policies)
```

Comment:
The cache policy can't be deleted if it is attached to a cache behavior. I believe deleting the distribution should be fine, and there is an existing terminator class for that, but we should set the age limit on cache policies to be a bit longer than distributions to minimize errors in the logs. The default age limit is 20 minutes. I would set this to 30 minutes. Here's an example of setting an age limit: https://github.com/mattclay/aws-terminator/blob/aaa0c2ba1e668cf68a0e0ee122c68909f5412830/aws/terminator/application_security.py#L8-L10

The same applies for the origin request policy.

Reply:
Applied to both policies; thank you for the recomendation.